src/applications/resources-and-support/components/SearchBar.jsx

```diff
@@ -18,7 +18,7 @@ export default function SearchBar({ onSearch, userInput, onInputChange }) {
         onSubmit={isGlobalSearch ? null : onSubmit}
         id="resources-support-search"
         method="get"
-        action={isGlobalSearch ? '/search' : '/resources/search'}
+        action={isGlobalSearch ? '/search/' : '/resources/search/'}
```

Comment:
You could use my constant here for `/resources/search/` if you want! 🙂 https://github.com/department-of-veterans-affairs/vets-website/pull/14865/files#diff-ef6a912a8c059f7814bf5b6bd82acd8ea06d150a6235a28b4d59289e890d640eR1

Reply:
Might be nice to put `/search/` in that constants file too, maybe calling it something like `VAGOV_SEARCH_PATHNAME`? That's a terrible name but you get the point 😅

Reply:
Oh yeah I suppose we could import the `manifest.json` from the different apps...good idea. I'll update